scripts/jenkins/ardana/ansible/deploy-cloud.yml

```diff
@@ -39,6 +39,32 @@
 
   tasks:
     - block:
+        - name: Remove versioned features from input model when not enabled
```

Comment:
This was moved to the input_model_generator and input_model_clone roles (see 1), you should not add it back here.

1. https://github.com/SUSE-Cloud/automation/commit/63b4bf84f1d974366d726f0613e078aa1a639464#diff-366adbdcd3a3db4bee004b373d3025d9